### PR TITLE
fix: persistent-breakpoints.nvim lazy loading

### PR DESCRIPTION
**Pull request by jay-babu**

### CODE REVIEW

**Comment by jay-babu** [comment]
https://github.com/Weissle/persistent-breakpoints.nvim/issues/15

**Comment by luxus** [comment]
is bufreadpost the right one?

**Comment by Uzaaft** [comment]
> is bufreadpost the right one?

See the issue he refered to.

**Comment by jay-babu** [comment]
It can also load after dap. Line 6 is already bufreadpost.

**Comment by Uzaaft** [comment]
> It can also load after dap. Line 6 is already bufreadpost.

But wouldn't "user Astrofile" be a better event to trigger it at?

File: lua/astrocommunity/debugging/persistent-breakpoints-nvim/init.lua [hunk 1]
```diff
@@ -1,5 +1,6 @@
 return {
   "Weissle/persistent-breakpoints.nvim",
+  event = "BufReadPost",
   opts = function(_, opts)
     return require("astronvim.utils").extend_tbl(opts, {
       load_breakpoints_event = { "BufReadPost" },
```
Comment (jay-babu) [inline lua/astrocommunity/debugging/persistent-breakpoints-nvim/init.lua:6]: I kept it the same as this. Unsure what is better. I'll leave up to your hands

Comment (Uzaaft) [reply lua/astrocommunity/debugging/persistent-breakpoints-nvim/init.lua:6]: Let's stick with this. :)

**Review by Uzaaft — APPROVED** [review]

**Pull request merged**